### PR TITLE
Javascript redo javascript2 week2

### PR DESCRIPTION
**Pull request by ShravyaGurmitkal**

### CODE REVIEW

**Review by semih1239 — APPROVED** [review]
You did a great job @ShravyaGurmitkal 💪 I liked your homework and I did not see any problem that I can recommend to you. Also, you did your homework in a very organized and understandable way. Congratulations 🙂

File: javascript/javascript2/week2/hyf-bayredo/main.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        products.sort((firstItem, secondItem) => secondItem.price - firstItem.price);
+    } else {
+        products.sort((firstItem, secondItem) => secondItem.rating - firstItem.rating);
+    }
+}
+
+//shopping cart
+const totalPrice = products.reduce((totalSum, product) => totalSum + product.price, 0);
+const totalPriceMarkupValue = document.getElementById('totalPriceMarkup');
+totalPriceMarkupValue.innerHTML += totalPrice;
```
Comment (seenachacko) [inline javascript/javascript2/week2/hyf-bayredo/main.js:80]: Goog job , Clear code

**Review by seenachacko — APPROVED** [review]
Good job @ShravyaGurmitkal

**Comment by ShravyaGurmitkal** [comment]
Thanks @semih1239  and @seenachacko

**Pull request merged**